src/TestReactors.jl

```diff
@@ -4,36 +4,36 @@ using Sundials
 
 @testset "Test Reactors" begin
 
-phaseDict = readinput("../src/testing/liquid_phase.rms")
```

Comment:
This is going to create nasty merge conflicts with other PRs can we hold off on this until right before v1.0.0?